frontend/public/i18n.js

```diff
@@ -59,6 +59,9 @@ i18n
       'rhoas-plugin',
       'topology',
       'tour',
+      'user',
```

Comment:
I guess these lines don't need to be added now that you're putting everything into `public`?

Reply:
Removed user namespace. Thanks.

Reply:
These lines were not added by me. Please ask the owner to remove it. Thanks.

Reply:
I will remove these lines. Thanks.